### ISSUE TITLE
 Is this project still maintained?

### ISSUE THREAD

**Issue by Malgosiatobiasz1** [Inform-Software/sonar-groovy#179]

**Comment by TobiX** [comment]
Not really. It's in a "works for my usecase" state - I honestly don't have time to work on it currently. I'll be available to review and merge pull requests and maybe push some releases to the SonarQube update center, but I don't have the time to debug issues which don't affect me or my colleagues.

Since we are still on SonarQube LTS (8.9), issues with newer SonarQube versions don't affect us right now, so I have no priority to fix them proactively.

**Comment by Malgosiatobiasz1** [comment]
Of course I understand, thank you very much for the information.
